src/Jackett.Common/Definitions/insanetracker.yml

```diff
@@ -52,6 +52,8 @@
       username: "{{ .Config.username }}"
       password: "{{ .Config.password }}"
       notsecure: "1"
+      x: "24"
```

Comment:
are you sure x and y are required, they do not appear to be part of the login form?